README.md

```diff
@@ -371,6 +371,30 @@ nylas.messages.search('Hey!').then(messages => console.log(messages));
 
 ```
 
+Creating and Updating Webhooks
+-----
+You can programmatically create, read, update and delete webhooks.
+
+```javascript
+// create a new webhook
+const newWebhook = Nylas.webhooks.build({
+  callbackUrl: 'https://wwww.myapp.com/webhook',
+  state: 'active',
+  triggers: ['event.created', 'event.updated'],
+  });
+newWebhook.save().then(webhook => console.log(webhook.id));
+
+// get and update an existing webhook
```

Comment:
the readme might also be a good place to explain that you can only update state, if they want to update anything else they'll have to make a new post.